frontend/src/Header.js

```diff
@@ -12,6 +12,8 @@ function Header() {
 
     const profileClass = window.location.pathname.match(/.profile/) ? "active" : "";
     const customersClass = window.location.pathname === "/customers" ? "active" : "";
+    const categoryClass = window.location.pathname.match(/.category/) ? "active" : "";
+    const itemClass = window.location.pathname.match(/.item/) ? "active" : "";
```

Comment:
We can create a function with one param for this statement and replace all the above lines with it.

Reply:
done